.gitignore

```diff
@@ -20,3 +21,17 @@ package.json
 binding.gyp
 READMEFIRST
 npm-debug.log
+
+# lua files
+src/lua/mraaLUA_wrap.cxx
```

Comment:
I'm kind of confused why all these are needed, if you use a build/ directory then everything will get ignored. Let's keep this stuff out unless there's a good reason